exonum-node/src/lib.rs

```diff
@@ -511,6 +515,7 @@ impl NodeRole {
 
 impl NodeHandler {
     /// Creates `NodeHandler` using specified `Configuration`.
+    #[allow(clippy::too_many_arguments)]
```

Comment:
Imo the time for `NodeHandlerBuilder` is come.

Reply:
`NodeHandler` is not public though, so IMO the priority of introducing the builder is not that high 🙃.